src/main/webapp/css/transition.css

```diff
@@ -0,0 +1,130 @@
+/* Loading Screen Styling */
```

Comment:
These should be sectioned off like the previous ones with ==========================================================================

and without the comment indicating the end

Reply:
done